application/cms/views.py

```diff
@@ -1002,7 +1002,7 @@ def list_measure_page_versions(topic, subtopic, measure):
     measures.sort(reverse=True)
     if not measures:
         return redirect(url_for("cms.subtopic", topic=topic, subtopic=subtopic))
-    measure_title = measures[0].title if measures else ""
+    measure_title = "".join(measure.title for measure in measures if measure.latest)
```

Comment:
@samuelhwilliams feels a little idiosyncratic to be joining the titles of all the 'latest' measures in the array when you're only expecting one?

Could you do something like:
```python
measure_title = next(measure for measure in measures if measure.latest).title
```

(Possibly not much more readable. I was expecting to find something like the `detect` method in Ruby which would allow you to do this: 

```ruby
measures.detect(&:latest).title
```
)

Reply:
Yeah I haven't found the cleanest way of retrieving one item from a list really.

This way is preferable to yours as it will collapse to a blank string if this list is empty - yours will raise an attribute error after trying to access 'title' on `None`. It's a trivial problem that feels weirdly tricky in Python, but I haven't put much thought into it tbh, after finding the above solution that "works".

Reply:
Everything I've now read is suggesting your way of doing it, so let's go with that. 👍

Reply:
@samuelhwilliams I think throwing an error in this scenario would be ok though, as there should always be at least one 'latest' measure in the list? A blank title feels like an error.

Reply:
Yeah, makes sense.

Reply:
@samuelhwilliams heh, I only found the `next` method by googling too. Was surprised that Python doesn't seem to have `map`, `select` and `select_first` (or similar) methods on Arrays...?